doc/ova/2.0.0/PI-6/CORTX_on_Open_Virtual_Appliance_PI-6.rst

```diff
@@ -0,0 +1,78 @@
+For our Japanese community, this document has been translated and is available `here <https://qiita.com/Taroi_Japanista/items/0ac03f55dce3f7433adf>`_.
+
+***********************
+Recommended Hypervisors
+***********************
+All of the following hypervisors should work: `VMware ESX Server <https://www.vmware.com/products/esxi-and-esx.html>`_,
+`VMware vSphere <https://www.vmware.com/products/vsphere.html>`_,
+`VMware Workstation <https://www.vmware.com/products/workstation-pro.html>`_,
+`VMware Fusion <https://www.vmware.com/in/products/fusion/fusion-evaluation.html>`_
+
+**Important**: If you are running the VM in any of the VMWare hypervisors, it is not recommended to use VMware Tools, as CORTX may break due to kernel dependencies. For the same reason, please do not update the operating system in the image as that also might cause it to fail.
+
+**Specifications:**
+
+- CORTX OVA image is created with following minimum configuration:
+
+  - RAM: 10GB
+  - Processor: 8
+  - NIC: 1
+  - OS Disk: 1 disk of 20GB
+  - Data Disks: 3 disks of 5GB each
+
+**Prerequisites:**
+
+- Download the `CORTX OVA <https://cortxova.s3.us-west-2.amazonaws.com/ova-2.0.0-713.ova>`_ from `our release page <https://github.com/Seagate/cortx/releases/latest>`_.
+- To perform the S3 IO operations,refer to these instructions in `S3 IO Operations <https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md>`_.
+- Import the OVA image using the instruction provided in  to `Importing the OVA document <https://github.com/Seagate/cortx/blob/main/doc/Importing_OVA_File.rst>`_.
+- Ensure that the Virtualization platform has internet connectivity:
+   
+  - For VMware related troubleshooting, please refer to `VM Documents <https://docs.vmware.com/en/VMware-vSphere/index.html>`_. 
+  - If on the VMware WorkStation, you do not see an IPv4 network configured, then update virtual networking configuration. See `troubleshooting virtual network <https://github.com/Seagate/cortx/blob/main/doc/troubleshoot_virtual_network.rst>`_.
+
+**********
+Procedure
+**********
+
+#. Open the VM console, and login with the below credentials.
+
+   * Username: cortx 
+   * Password: opensource!
+  
+#. Become the **root** user by running this:
+   
+   ::
+   
+       sudo su -
+       
+   **Note** Wait for 5 - 10 mins till the cluster services are started.
+   
+#. Run the following command to check the health of CORTX using hctl by running this command:
+
+   ::
+
+       hctl status
+
+ 
+#. To perform the S3 IO operations, refer the instructions in `S3 IO operations <https://github.com/Seagate/cortx/blob/main/doc/ova/2.0.0/PI-6/S3_IO_Operations.md>`_.
+
+#. BOOM. You're all done and you're AWESOME. 
+
+   Thanks for checking out the CORTX system; we hope you liked it. Hopefully you'll stick around and participate in our community and help make it even better.
+
+ 
+
+***************
+Troubleshooting
```

Comment:
remove Troubleshooting section as not required for now

Reply:
Done